nibabel/gifti/gifti.py

```diff
@@ -453,7 +453,7 @@ def add_gifti_data_array(self, dataarr):
         if isinstance(dataarr, GiftiDataArray):
             self.darrays.append(dataarr)
         else:
-            print("dataarr paramater must be of tzpe GiftiDataArray")
+            raise ValueError("dataarr paramater must be of type GiftiDataArray")
 
```

Comment:
Good call on the style and error type. Fixed, and scrubbed the code for other `ValueError`s that should be `TypeError`s. Tests running.